test/functional/test-document-submit.js

```diff
@@ -46,6 +46,43 @@ describe('test-document-submit onDocumentFormSubmit_', () => {
     sandbox.restore();
   });
 
+  describe('installGlobalSubmitListenerForDoc', () => {
+    let ampDoc;
+    let headNode;
+    let rootNode;
+    beforeEach(() => {
+      headNode = document.createElement('head');
+      rootNode = document.createElement('html');
+      ampDoc = {
+        getHeadNode: () => headNode,
+        getRootNode: () => rootNode,
+      };
+    });
+
+    afterEach(() => sandbox.reset());
```

Comment:
Shouldn't be necessary if you don't create the sandbox in `beforeEach()`.

Reply:
removed